packages/cli/src/utils/init.js

```diff
@@ -28,6 +30,70 @@ const initApp = async (path, createFunc) => {
   await removeDir(tempAppDir);
 };
 
+const testAuth = authType => {
+  // todo: oauth1 trello url?
```

Comment:
Yeah, we can use Trello for OAuth1. Or, we can add OAuth1 support to https://auth-json-server.zapier-staging.com and use that. But I'd just use what we have (Trello) for now.